ui/src/app/views/workspace/component.html

```diff
@@ -1,10 +1,16 @@
 <div *ngIf="!notFound else notFoundMessage">
   <div *ngIf="!workspaceLoading; else workspacePageLoading" style="display: flex;">
     <h2>{{workspace.name}}</h2>
-    <span *ngIf="hasPermission()">
-      <clr-icon class="editing-icon is-solid" shape="pencil" (click)="edit()"></clr-icon>
-    </span>
-    <clr-icon class="sharing-icon is-solid" shape="users" (click)="share()"></clr-icon>
+    <div class="icon-area">
+      <clr-icon *ngIf="hasWritePermission()"
+          class="editing-icon is-solid" shape="pencil"
+          (click)="edit()"></clr-icon>
+      <clr-icon class="sharing-icon is-solid" shape="users" (click)="share()"></clr-icon>
+      <clr-icon *ngIf="hasOwnerPermission()"
+          class="deleting-icon is-solid"
+          shape="trash" (click)="delete()"></clr-icon>
+      <span *ngIf="deleting" class="icon-notify spinner spinner-sm"></span>
+    </div>
```

Comment:
I think disabled definitely feels better, so I'm doing that, but punting on tooltips till after Feb 1st, since we have a lot of disabled items that don't have tooltips right now, and that feels like something we can go through and clean up after security freeze.